flocker/control/test/test_persistence.py

```diff
@@ -123,6 +134,24 @@ def saved_again(_):
         d.addCallback(saved_again)
         return d
 
+    @validate_logging(
+        lambda test, logger:
+        test.assertEqual(len(logger.flush_tracebacks(ZeroDivisionError)), 1))
```

Comment:
I guess it's not quite as bad as I first though since you're only flushing `ZeroDivisionError`.  Any other unexpected traceback written by this test won't get flushed, will fail the test, and will get reported in the test summary.  So yea okay.